# School Location Choice Model

In this notebook, a simple logit location choice model is used to predict university campus choice. Data from the 2015 StudentMoveTO data is used. The model is run separately on each of the 13 segments we have previously outlined.

First, we install Biogeme and Pandas, and load the original data.

In [1]:
import biogeme.database as db
import biogeme.biogeme as bio
import biogeme.models as models
from biogeme.expressions import Beta, DefineVariable
import biogeme.messaging as msg
import pandas as pd

hh_df = pd.read_csv('../Data/SMTO_2015/SMTO_2015_Households.csv')
ps_df = pd.read_csv('../Data/SMTO_2015/SMTO_2015_Respondents.csv')

Now, we prepare a dataframe which will be used as input for Biogeme by selecting and manipulating the relevant columns from the data. We use a function to convert information about Level, Status, Years of Involvement, and Living Arrangement into a column containing the respondent's Segment.

In [2]:
def info_to_segment(x):
    if x.Level == 'Other':
        return 0
    elif x.Level == 'UG':
        return (0 if x.Status == 'FT' else 2) + (not x.Family) + (1 if x.Years == 0 else 5)
    else:
        return 9 + (0 if x.Status == 'FT' else 2) + (not x.Family)

# Load relevant columns
df = ps_df[['pscampusattend', 'personstatusgrad', 'personstatustime', 'psuniversityinvolvednumyears']]
df = df.join(hh_df[['HmTTS2006', 'hhlivingsituation']])
df = df.rename(columns={'HmTTS2006': 'HomeZone', 'pscampusattend': 'Campus', 'hhlivingsituation': 'Family', 
                       'personstatusgrad': 'Level', 'personstatustime': 'Status', 'psuniversityinvolvednumyears': 'Years'})
df = df.dropna() # Remove rows with missing data

# Convert Campus column to numerical column
campus_name_to_num = {"Downtown Toronto (St. George)": 0, "Scarborough (UTSC)": 1, "Mississauga (UTM)": 2,
                      "Keele": 3, "Glendon": 4, "RyersonU": 5, "OCADu": 6}
df.replace({'Campus': campus_name_to_num}, inplace=True)
df['HomeZone'] = pd.to_numeric(df['HomeZone'], downcast='signed')
df['Family'] = (df['Family'] == 'Live with family/parents')*1
df['Segment'] = df.apply(info_to_segment, axis=1)
df['Available'] = 1 # All campuses available to all students

df['Segment'].value_counts()

5     4914
6     2438
10    2271
1     2012
2     1024
9      872
0      318
7      315
8      279
12     243
11     109
3       69
4       46
Name: Segment, dtype: int64

Keeping only the relevant columns, we have:

In [3]:
df = df[['Campus', 'HomeZone', 'Segment', 'Available']]
df

Campus  HomeZone  Segment  Available
0           1       261        5          1
1           0        71       10          1
2           0      3714        5          1
3           0        74        6          1
4           0        71       10          1
...       ...       ...      ...        ...
15221       3       212        2          1
15222       3       233        5          1
15223       3        95        6          1
15224       3      2221        5          1
15225       3      3686        5          1

[14910 rows x 4 columns]

Now we must load information about the distance between each respondent's home zone and each campus. First, we load the data from the LoS matrix.

In [4]:
# Dataframe with walk distances
df_path = pd.read_csv('../../LoS/Walk_Distances.csv')
origins = list(set(list(df_path['Origin'])))
dists = list(df_path['Data'])

Now, we declare a function to look up the walk distance given an origin-destination pair.

In [5]:
not_found = set()
# Function for distance lookup
def find_distance(origin, destination):
    try:
        i = origins.index(origin)
    except ValueError:
        not_found.add(origin)
        return 0
    try:
        j = origins.index(destination)
    except ValueError:
        not_found.add(destination)
        return 0
    return dists[i*2392 + j] / 1000

We run this function for each row on each school zone, creating seven new columns.

In [6]:
# List of campus' TTS zones from Joven's MOE data
campus_zones = [69, 566, 3631, 391, 225, 38, 67]

# Load distances in dataframe
for i in range(7):
    df["Dist" + str(i)] = df['HomeZone'].apply(lambda x: find_distance(x, campus_zones[i]))
len(not_found)

127

127 zones were not found as they are outside the GTHA. Let's see what our dataframe looks like now:

In [7]:
df

Campus  HomeZone  Segment  Available      Dist0     Dist1     Dist2  \
0           1       261        5          1  10.256060  14.88098  29.20657   
1           0        71       10          1   1.132351  23.03920  19.64290   
2           0      3714        5          1  23.319230  45.63271   4.51742   
3           0        74        6          1   0.699414  24.11954  19.43932   
4           0        71       10          1   1.132351  23.03920  19.64290   
...       ...       ...      ...        ...        ...       ...       ...   
15221       3       212        2          1   7.120260  19.01731  23.76046   
15222       3       233        5          1  15.917590  12.03644  32.96591   
15223       3        95        6          1   2.783940  25.09743  17.84462   
15224       3      2221        5          1  23.379880  26.15476  37.35434   
15225       3      3686        5          1  35.537980  53.96100  16.14689   

          Dist3      Dist4      Dist5      Dist6  
0      22.59214   9.218413   9.580635  11.241730  
1      15.87906  11.211150   2.675173   2.723838  
2      28.58045  32.555200  24.964000  23.686150  
3      16.81186  12.830410   2.314008   1.541276  
4      15.87906  11.211150   2.675173   2.723838  
...         ...        ...        ...        ...  
15221  14.19530   5.732956   6.553092   8.193741  
15222  17.68772   6.019180  15.242170  16.903260  
15223  15.53600  12.829600   4.733398   3.979057  
15224  13.30458  15.379040  23.250580  24.518920  
15225  34.80651  40.034760  36.584780  34.882980  

[14910 rows x 11 columns]

We are now ready to run our model! We do this 13 times, each time keeping only one segment.

In [8]:
for i in range(13):
    temp_df = df.copy()
    database = db.Database("SMTO", temp_df)
    globals().update(database.variables)
    database.remove(Dist0 == 0) # Remove unknown distances
    
    database.remove(Segment != i)
    
    # Beta initialization: (name, value, lowerbound, upperbound, status, desc='')
    # Status 0 if estimated, 1 if maintained - reference choice should be 1
    ASC_SG = Beta('ASC_SG', 0, None, None, 0)
    ASC_SC = Beta('ASC_SC', 0, None, None, 0)
    ASC_MI = Beta('ASC_MI', 0, None, None, 0)
    ASC_YK = Beta('ASC_YK', 0, None, None, 0)
    ASC_YG = Beta('ASC_YG', 0, None, None, 1)
    ASC_RY = Beta('ASC_RY', 0, None, None, 0)
    ASC_OC = Beta('ASC_OC', 0, None, None, 0)
    B_DIST = Beta('B_DIST', 0, None, None, 0)

    # Variables: from columns in database
    AV = DefineVariable('AV', Available, database)
    SG_DIST = DefineVariable('SG_DIST', Dist0, database)
    SC_DIST = DefineVariable('SC_DIST', Dist1, database)
    MI_DIST = DefineVariable('MI_DIST', Dist2, database)
    YK_DIST = DefineVariable('YK_DIST', Dist3, database)
    YG_DIST = DefineVariable('YG_DIST', Dist4, database)
    RY_DIST = DefineVariable('RY_DIST', Dist5, database)
    OC_DIST = DefineVariable('OC_DIST', Dist6, database)

    # Utility Functions: note ASC_SG is 0
    V0 = ASC_SG + B_DIST * SG_DIST
    V1 = ASC_SC + B_DIST * SC_DIST
    V2 = ASC_MI + B_DIST * MI_DIST
    V3 = ASC_YK + B_DIST * YK_DIST
    V4 = ASC_YG + B_DIST * YG_DIST
    V5 = ASC_RY + B_DIST * RY_DIST
    V6 = ASC_OC + B_DIST * OC_DIST

    V  = {0: V0, 1: V1, 2: V2, 3: V3, 4: V4, 5: V5, 6: V6}
    av = {0: AV, 1: AV, 2: AV, 3: AV, 4: AV, 5: AV, 6: AV}
    
    logprob = models.loglogit(V, av, Campus)
    logger = msg.bioMessage()
    logger.setDebug()
    biogeme = bio.BIOGEME(database, logprob, numberOfThreads=1)
    biogeme.modelName = "SMTO_Segmented_Output/SMTO_Campus_Choice_Segment_" + str(i)
    results = biogeme.estimate(saveIterations=True)
    biogeme.createLogFile()
    betas = results.getBetaValues()
    for k,v in betas.items():
        print(f"{k:10}=\t{v:.3g}")
    pandasResults = results.getEstimatedParameters()
    print(pandasResults)

[10:32:23] < General >   Remove 10 unused variables from the database as only 9 are used.
[10:32:23] < General >   Log likelihood (N=309):  -601.2862
[10:32:23] < General >   Minimize with tol 1e-07
[10:32:23] < Debug >     ASC_MI:          0
[10:32:23] < Debug >     ASC_OC:          0
[10:32:23] < Debug >     ASC_RY:          0
[10:32:23] < Debug >     ASC_SC:          0
[10:32:23] < Debug >     ASC_SG:          0
[10:32:23] < Debug >     ASC_YK:          0
[10:32:23] < Debug >     B_DIST:          0
[10:32:23] < General >   Log likelihood (N=309):  -601.2862 Gradient norm:      2e+03  
[10:32:23] < Debug >     ASC_MI: -0.02249692
[10:32:23] < Debug >     ASC_OC: -0.02623158
[10:32:23] < Debug >     ASC_RY: 0.07771662
[10:32:23] < Debug >     ASC_SC: -0.02311936
[10:32:23] < Debug >     ASC_SG: 0.01858441
[10:32:23] < Debug >     ASC_YK: -0.003823587
[10:32:23] < Debug >     B_DIST: -0.9959274
[10:32:23] < General >   Log likelihood (N=309):  -1887.277 Gradient norm:      2e+03  
[10:

[10:32:23] < Debug >     ASC_MI: -0.3160336
[10:32:23] < Debug >     ASC_OC:   -2.03486
[10:32:23] < Debug >     ASC_RY:   2.548025
[10:32:23] < Debug >     ASC_SC: -0.4397623
[10:32:23] < Debug >     ASC_SG:   1.697767
[10:32:23] < Debug >     ASC_YK:   1.193664
[10:32:23] < Debug >     B_DIST: -0.09468609
[10:32:23] < General >   Log likelihood (N=309):    -357.81 Gradient norm:          3  
[10:32:23] < Debug >     ASC_MI: -0.3154496
[10:32:23] < Debug >     ASC_OC:   -2.03704
[10:32:23] < Debug >     ASC_RY:   2.550598
[10:32:23] < Debug >     ASC_SC: -0.4406585
[10:32:23] < Debug >     ASC_SG:    1.69716
[10:32:23] < Debug >     ASC_YK:   1.193871
[10:32:23] < Debug >     B_DIST: -0.09499147
[10:32:23] < General >   Log likelihood (N=309):  -357.8094 Gradient norm:          1  
[10:32:23] < Debug >     ASC_MI: -0.3156752
[10:32:23] < Debug >     ASC_OC:  -2.034841
[10:32:23] < Debug >     ASC_RY:   2.550071
[10:32:23] < Debug >     ASC_SC: -0.4399891
[10:32:23] < Debug >     ASC_S

[10:32:24] < General >   Results saved in file SMTO_Segmented_Output/SMTO_Campus_Choice_Segment_0.html
[10:32:24] < General >   Results saved in file SMTO_Segmented_Output/SMTO_Campus_Choice_Segment_0.pickle
[10:32:24] < General >   Log file created: SMTO_Segmented_Output/SMTO_Campus_Choice_Segment_0.log
ASC_MI    =	-0.255
ASC_OC    =	-1.8
ASC_RY    =	2.63
ASC_SC    =	-0.289
ASC_SG    =	1.77
ASC_YK    =	1.27
B_DIST    =	-0.0949
           Value   Std err    t-test       p-value  Rob. Std err  Rob. t-test  \
ASC_MI -0.255495  0.487220 -0.524394  6.000047e-01      0.459624    -0.555878   
ASC_OC -1.799090  0.770065 -2.336283  1.947648e-02      0.774038    -2.324292   
ASC_RY  2.629828  0.313975  8.375919  0.000000e+00      0.318148     8.266044   
ASC_SC -0.288939  0.500975 -0.576754  5.641055e-01      0.516059    -0.559896   
ASC_SG  1.774762  0.325967  5.444611  5.191874e-08      0.328226     5.407133   
ASC_YK  1.267303  0.347069  3.651449  2.607649e-04      0.344103     3.682923   
B

[10:32:25] < Debug >     ASC_MI:  0.9266359
[10:32:25] < Debug >     ASC_OC: -0.2498007
[10:32:25] < Debug >     ASC_RY:   1.883655
[10:32:25] < Debug >     ASC_SC:  0.9516079
[10:32:25] < Debug >     ASC_SG:   1.971169
[10:32:25] < Debug >     ASC_YK:   1.895051
[10:32:25] < Debug >     B_DIST: -0.06579851
[10:32:25] < General >   Log likelihood (N=1984):  -3181.982 Gradient norm:      9e+01  
[10:32:25] < Debug >     ASC_MI:   1.046445
[10:32:25] < Debug >     ASC_OC: 0.09268343
[10:32:25] < Debug >     ASC_RY:   2.023957
[10:32:25] < Debug >     ASC_SC:    1.04279
[10:32:25] < Debug >     ASC_SG:   2.048052
[10:32:25] < Debug >     ASC_YK:   1.972311
[10:32:25] < Debug >     B_DIST: -0.06574397
[10:32:25] < General >   Log likelihood (N=1984):  -3177.143 Gradient norm:      4e+01  
[10:32:25] < Debug >     ASC_MI:   1.268982
[10:32:25] < Debug >     ASC_OC:  0.2116194
[10:32:25] < Debug >     ASC_RY:   2.240821
[10:32:25] < Debug >     ASC_SC:   1.279487
[10:32:25] < Debug >     ASC

[10:32:26] < General >   Results saved in file SMTO_Segmented_Output/SMTO_Campus_Choice_Segment_1.html
[10:32:26] < General >   Results saved in file SMTO_Segmented_Output/SMTO_Campus_Choice_Segment_1.pickle
[10:32:26] < General >   Log file created: SMTO_Segmented_Output/SMTO_Campus_Choice_Segment_1.log
ASC_MI    =	0.976
ASC_OC    =	0.203
ASC_RY    =	2.04
ASC_SC    =	1.09
ASC_SG    =	1.99
ASC_YK    =	2.01
B_DIST    =	-0.0657
           Value   Std err     t-test       p-value  Rob. Std err  \
ASC_MI  0.975690  0.145308   6.714610  1.885714e-11      0.142198   
ASC_OC  0.202513  0.167988   1.205523  2.280015e-01      0.169032   
ASC_RY  2.041118  0.129606  15.748657  0.000000e+00      0.130574   
ASC_SC  1.094376  0.142888   7.658997  1.865175e-14      0.142966   
ASC_SG  1.992882  0.129779  15.355975  0.000000e+00      0.130782   
ASC_YK  2.014415  0.128623  15.661412  0.000000e+00      0.128758   
B_DIST -0.065668  0.003229 -20.339343  0.000000e+00      0.003009   

        Rob. t-te

[10:32:27] < Debug >     ASC_MI: -0.2628471
[10:32:27] < Debug >     ASC_OC:  -1.293745
[10:32:27] < Debug >     ASC_RY:  0.1706979
[10:32:27] < Debug >     ASC_SC:  -0.209339
[10:32:27] < Debug >     ASC_SG:   1.073104
[10:32:27] < Debug >     ASC_YK:   1.327118
[10:32:27] < Debug >     B_DIST: -0.1816967
[10:32:27] < General >   Log likelihood (N=942):  -918.4274 Gradient norm:      1e+02  
[10:32:28] < Debug >     ASC_MI: -0.2797418
[10:32:28] < Debug >     ASC_OC:  -1.370423
[10:32:28] < Debug >     ASC_RY:  0.1817907
[10:32:28] < Debug >     ASC_SC: -0.2256425
[10:32:28] < Debug >     ASC_SG:   1.145855
[10:32:28] < Debug >     ASC_YK:   1.435015
[10:32:28] < Debug >     B_DIST: -0.1890218
[10:32:28] < General >   Log likelihood (N=942):  -916.9765 Gradient norm:      2e+01  
[10:32:28] < Debug >     ASC_MI: -0.2823366
[10:32:28] < Debug >     ASC_OC:  -1.381952
[10:32:28] < Debug >     ASC_RY:   0.182258
[10:32:28] < Debug >     ASC_SC: -0.2286111
[10:32:28] < Debug >     ASC_SG:

[10:32:28] < Debug >     ASC_MI:  0.2346027
[10:32:28] < Debug >     ASC_OC: -0.6700525
[10:32:28] < Debug >     ASC_RY:  0.6475299
[10:32:28] < Debug >     ASC_SC: 0.05117624
[10:32:28] < Debug >     ASC_SG:   1.639944
[10:32:28] < Debug >     ASC_YK:   1.925312
[10:32:28] < Debug >     B_DIST: -0.1878129
[10:32:28] < General >   Log likelihood (N=942):  -912.3758 Gradient norm:          7  
[10:32:28] < Debug >     ASC_MI:  0.2363534
[10:32:28] < Debug >     ASC_OC: -0.6669322
[10:32:28] < Debug >     ASC_RY:  0.6488721
[10:32:28] < Debug >     ASC_SC: 0.05335645
[10:32:28] < Debug >     ASC_SG:   1.639027
[10:32:28] < Debug >     ASC_YK:    1.92552
[10:32:28] < Debug >     B_DIST: -0.1881075
[10:32:28] < General >   Log likelihood (N=942):  -912.3742 Gradient norm:        0.2  
[10:32:28] < Debug >     ASC_MI:  0.2362107
[10:32:28] < Debug >     ASC_OC: -0.6673257
[10:32:28] < Debug >     ASC_RY:  0.6484957
[10:32:28] < Debug >     ASC_SC: 0.05330475
[10:32:28] < Debug >     ASC_SG:

[10:32:29] < Debug >     ASC_SC:  0.2133892
[10:32:29] < Debug >     ASC_SG:   2.300196
[10:32:29] < Debug >     ASC_YK:   3.378033
[10:32:29] < Debug >     B_DIST: -0.05267982
[10:32:29] < General >   Log likelihood (N=68):  -93.67336 Gradient norm:      4e-05  
[10:32:29] < Debug >     ASC_MI:   1.868013
[10:32:29] < Debug >     ASC_OC:  0.7151336
[10:32:29] < Debug >     ASC_RY:   2.913241
[10:32:29] < Debug >     ASC_SC:  0.2133895
[10:32:29] < Debug >     ASC_SG:   2.300195
[10:32:29] < Debug >     ASC_YK:   3.378032
[10:32:29] < Debug >     B_DIST: -0.05267983
[10:32:29] < General >   Log likelihood (N=68):  -93.67336 Gradient norm:      1e-05  
[10:32:29] < Debug >     ASC_MI:   1.868012
[10:32:29] < Debug >     ASC_OC:  0.7151332
[10:32:29] < Debug >     ASC_RY:   2.913241
[10:32:29] < Debug >     ASC_SC:  0.2133895
[10:32:29] < Debug >     ASC_SG:   2.300195
[10:32:29] < Debug >     ASC_YK:   3.378032
[10:32:29] < Debug >     B_DIST: -0.05267984
[10:32:29] < General >   Log li

[10:32:31] < Debug >     ASC_MI: -0.5052631
[10:32:31] < Debug >     ASC_OC:  -0.519895
[10:32:31] < Debug >     ASC_RY:  0.7146996
[10:32:31] < Debug >     ASC_SC:   0.673249
[10:32:31] < Debug >     ASC_SG:   1.255524
[10:32:31] < Debug >     ASC_YK:   1.648118
[10:32:31] < Debug >     B_DIST: -0.06644001
[10:32:31] < General >   Log likelihood (N=43):  -67.58501 Gradient norm:     0.0007  
[10:32:31] < Debug >     ASC_MI: -0.5052799
[10:32:31] < Debug >     ASC_OC: -0.5198865
[10:32:31] < Debug >     ASC_RY:  0.7146731
[10:32:31] < Debug >     ASC_SC:  0.6732476
[10:32:31] < Debug >     ASC_SG:   1.255513
[10:32:31] < Debug >     ASC_YK:   1.648094
[10:32:31] < Debug >     B_DIST: -0.06643946
[10:32:31] < General >   Log likelihood (N=43):  -67.58501 Gradient norm:      0.001  
[10:32:31] < Debug >     ASC_MI: -0.5053345
[10:32:31] < Debug >     ASC_OC:  -0.519865
[10:32:31] < Debug >     ASC_RY:  0.7146614
[10:32:31] < Debug >     ASC_SC:  0.6732634
[10:32:31] < Debug >     ASC_SG:

[10:32:33] < Debug >     ASC_MI:  -0.054454
[10:32:33] < Debug >     ASC_OC: -0.1016884
[10:32:33] < Debug >     ASC_RY:  0.1052943
[10:32:33] < Debug >     ASC_SC: -0.01655551
[10:32:33] < Debug >     ASC_SG:  0.1077707
[10:32:33] < Debug >     ASC_YK: 0.07742094
[10:32:33] < Debug >     B_DIST: -0.03892171
[10:32:33] < General >   Log likelihood (N=4859):  -8714.461 Gradient norm:      7e+03  
[10:32:33] < Debug >     ASC_MI: -0.1462682
[10:32:33] < Debug >     ASC_OC: -0.2739056
[10:32:33] < Debug >     ASC_RY:  0.2836383
[10:32:33] < Debug >     ASC_SC: -0.0439765
[10:32:33] < Debug >     ASC_SG:  0.2902324
[10:32:33] < Debug >     ASC_YK:  0.2083566
[10:32:33] < Debug >     B_DIST: -0.02854875
[10:32:33] < General >   Log likelihood (N=4859):  -8520.365 Gradient norm:      1e+04  
[10:32:33] < Debug >     ASC_MI: -0.2982422
[10:32:33] < Debug >     ASC_OC:  -0.558797
[10:32:33] < Debug >     ASC_RY:   0.579241
[10:32:33] < Debug >     ASC_SC: -0.08919041
[10:32:33] < Debug >     A

[10:32:34] < Debug >     ASC_MI:  0.3793679
[10:32:34] < Debug >     ASC_OC:     -1.305
[10:32:34] < Debug >     ASC_RY:   1.675146
[10:32:34] < Debug >     ASC_SC:   1.007979
[10:32:34] < Debug >     ASC_SG:   1.693457
[10:32:34] < Debug >     ASC_YK:   1.397602
[10:32:34] < Debug >     B_DIST: -0.06812591
[10:32:34] < General >   Log likelihood (N=4859):  -7666.797 Gradient norm:      4e+02  
[10:32:34] < Debug >     ASC_MI:  0.4379274
[10:32:34] < Debug >     ASC_OC:  -1.230936
[10:32:34] < Debug >     ASC_RY:   1.659104
[10:32:34] < Debug >     ASC_SC:  0.9704397
[10:32:34] < Debug >     ASC_SG:   1.681529
[10:32:34] < Debug >     ASC_YK:   1.381227
[10:32:34] < Debug >     B_DIST: -0.06881155
[10:32:34] < General >   Log likelihood (N=4859):  -7661.328 Gradient norm:      8e+02  
[10:32:34] < Debug >     ASC_MI:  0.5638462
[10:32:34] < Debug >     ASC_OC:  -1.122034
[10:32:34] < Debug >     ASC_RY:   1.674615
[10:32:34] < Debug >     ASC_SC:  0.9334701
[10:32:34] < Debug >     ASC

[10:32:35] < Debug >     ASC_MI:     1.2197
[10:32:35] < Debug >     ASC_OC:  0.1199052
[10:32:35] < Debug >     ASC_RY:   2.380138
[10:32:35] < Debug >     ASC_SC:   1.599361
[10:32:35] < Debug >     ASC_SG:   2.452194
[10:32:35] < Debug >     ASC_YK:   2.215911
[10:32:35] < Debug >     B_DIST: -0.06664791
[10:32:35] < General >   Log likelihood (N=4859):  -7588.431 Gradient norm:      7e+01  
[10:32:35] < Debug >     ASC_MI:   1.235464
[10:32:35] < Debug >     ASC_OC:  0.1570729
[10:32:35] < Debug >     ASC_RY:   2.411886
[10:32:35] < Debug >     ASC_SC:   1.625717
[10:32:35] < Debug >     ASC_SG:   2.460204
[10:32:35] < Debug >     ASC_YK:   2.240869
[10:32:35] < Debug >     B_DIST: -0.06673574
[10:32:35] < General >   Log likelihood (N=4859):  -7587.461 Gradient norm:      5e+01  
[10:32:35] < Debug >     ASC_MI:   1.236119
[10:32:35] < Debug >     ASC_OC:  0.1768944
[10:32:35] < Debug >     ASC_RY:   2.441566
[10:32:35] < Debug >     ASC_SC:   1.626977
[10:32:35] < Debug >     ASC

[10:32:37] < Debug >     ASC_MI:    1.22862
[10:32:37] < Debug >     ASC_OC:  0.1699242
[10:32:37] < Debug >     ASC_RY:   2.434356
[10:32:37] < Debug >     ASC_SC:   1.614266
[10:32:37] < Debug >     ASC_SG:   2.431201
[10:32:37] < Debug >     ASC_YK:   2.231168
[10:32:37] < Debug >     B_DIST: -0.06682574
[10:32:37] < General >   Log likelihood (N=4859):  -7586.608 Gradient norm:      0.003  
[10:32:37] < Debug >     ASC_MI:    1.22862
[10:32:37] < Debug >     ASC_OC:  0.1699237
[10:32:37] < Debug >     ASC_RY:   2.434356
[10:32:37] < Debug >     ASC_SC:   1.614266
[10:32:37] < Debug >     ASC_SG:   2.431201
[10:32:37] < Debug >     ASC_YK:   2.231168
[10:32:37] < Debug >     B_DIST: -0.06682573
[10:32:37] < General >   Log likelihood (N=4859):  -7586.608 Gradient norm:      0.003  
[10:32:37] < Debug >     ASC_MI:   1.228622
[10:32:37] < Debug >     ASC_OC:  0.1699227
[10:32:37] < Debug >     ASC_RY:   2.434358
[10:32:37] < Debug >     ASC_SC:   1.614269
[10:32:37] < Debug >     ASC

[10:32:39] < Debug >     ASC_MI: -0.2204501
[10:32:39] < Debug >     ASC_OC:   -1.19704
[10:32:39] < Debug >     ASC_RY:  0.2330485
[10:32:39] < Debug >     ASC_SC: 0.05651409
[10:32:39] < Debug >     ASC_SG:    1.19054
[10:32:39] < Debug >     ASC_YK:  0.8398209
[10:32:39] < Debug >     B_DIST: -0.1563638
[10:32:39] < General >   Log likelihood (N=2392):  -2672.149 Gradient norm:      8e+02  
[10:32:39] < Debug >     ASC_MI: -0.2487798
[10:32:39] < Debug >     ASC_OC:  -1.259964
[10:32:39] < Debug >     ASC_RY:  0.3558211
[10:32:39] < Debug >     ASC_SC: 0.09732246
[10:32:39] < Debug >     ASC_SG:    1.16792
[10:32:39] < Debug >     ASC_YK:   1.032751
[10:32:39] < Debug >     B_DIST: -0.1664135
[10:32:39] < General >   Log likelihood (N=2392):  -2656.298 Gradient norm:      3e+02  
[10:32:39] < Debug >     ASC_MI: -0.2506358
[10:32:39] < Debug >     ASC_OC:   -1.24196
[10:32:39] < Debug >     ASC_RY:   0.387596
[10:32:39] < Debug >     ASC_SC:  0.1172676
[10:32:39] < Debug >     ASC_S

[10:32:39] < Debug >     ASC_MI:  0.6625957
[10:32:39] < Debug >     ASC_OC: -0.0940455
[10:32:39] < Debug >     ASC_RY:   1.190328
[10:32:39] < Debug >     ASC_SC:   1.282579
[10:32:39] < Debug >     ASC_SG:   1.986916
[10:32:39] < Debug >     ASC_YK:   1.973214
[10:32:39] < Debug >     B_DIST: -0.1682861
[10:32:39] < General >   Log likelihood (N=2392):  -2614.477 Gradient norm:          6  
[10:32:39] < Debug >     ASC_MI:  0.6786823
[10:32:39] < Debug >     ASC_OC: -0.03295785
[10:32:39] < Debug >     ASC_RY:   1.217036
[10:32:39] < Debug >     ASC_SC:   1.305309
[10:32:39] < Debug >     ASC_SG:   2.031904
[10:32:39] < Debug >     ASC_YK:   2.029831
[10:32:39] < Debug >     B_DIST: -0.1684531
[10:32:39] < General >   Log likelihood (N=2392):  -2614.488 Gradient norm:      1e+01  
[10:32:39] < Debug >     ASC_MI:   0.670133
[10:32:39] < Debug >     ASC_OC: -0.06542313
[10:32:39] < Debug >     ASC_RY:   1.202842
[10:32:39] < Debug >     ASC_SC:   1.293229
[10:32:39] < Debug >     ASC

[10:32:42] < Debug >     ASC_SC:  0.6191073
[10:32:42] < Debug >     ASC_SG:   1.927181
[10:32:42] < Debug >     ASC_YK:   1.161907
[10:32:42] < Debug >     B_DIST: -0.07025896
[10:32:42] < General >   Log likelihood (N=310):  -480.6482 Gradient norm:      1e+01  
[10:32:42] < Debug >     ASC_MI:  -0.309224
[10:32:42] < Debug >     ASC_OC: -0.4123612
[10:32:42] < Debug >     ASC_RY:   1.104063
[10:32:42] < Debug >     ASC_SC:  0.7749932
[10:32:42] < Debug >     ASC_SG:   2.063818
[10:32:42] < Debug >     ASC_YK:   1.329097
[10:32:42] < Debug >     B_DIST: -0.06815975
[10:32:42] < General >   Log likelihood (N=310):  -479.6373 Gradient norm:      4e+01  
[10:32:42] < Debug >     ASC_MI: -0.06794492
[10:32:42] < Debug >     ASC_OC: -0.1776558
[10:32:42] < Debug >     ASC_RY:   1.136077
[10:32:42] < Debug >     ASC_SC:   1.028811
[10:32:42] < Debug >     ASC_SG:   2.310468
[10:32:42] < Debug >     ASC_YK:   1.541945
[10:32:42] < Debug >     B_DIST: -0.0756917
[10:32:42] < General >   Log 

[10:32:42] < Debug >     ASC_MI:  0.7620901
[10:32:42] < Debug >     ASC_OC:   0.692008
[10:32:42] < Debug >     ASC_RY:   1.719047
[10:32:42] < Debug >     ASC_SC:   1.434691
[10:32:42] < Debug >     ASC_SG:   2.683474
[10:32:42] < Debug >     ASC_YK:   2.013783
[10:32:42] < Debug >     B_DIST: -0.06783771
[10:32:42] < General >   Log likelihood (N=310):  -474.3812 Gradient norm:      0.005  
[10:32:42] < Debug >     ASC_MI:  0.7620518
[10:32:42] < Debug >     ASC_OC:  0.6919828
[10:32:42] < Debug >     ASC_RY:   1.719032
[10:32:42] < Debug >     ASC_SC:   1.434666
[10:32:42] < Debug >     ASC_SG:   2.683446
[10:32:42] < Debug >     ASC_YK:   2.013766
[10:32:42] < Debug >     B_DIST: -0.0678374
[10:32:42] < General >   Log likelihood (N=310):  -474.3812 Gradient norm:     0.0006  
[10:32:42] < Debug >     ASC_MI:   0.762046
[10:32:42] < Debug >     ASC_OC:  0.6919733
[10:32:42] < Debug >     ASC_RY:   1.719025
[10:32:42] < Debug >     ASC_SC:   1.434657
[10:32:42] < Debug >     ASC_SG

[10:32:43] < Debug >     ASC_OC:  0.1566693
[10:32:43] < Debug >     ASC_RY:   1.209126
[10:32:43] < Debug >     ASC_SC: 0.08913963
[10:32:43] < Debug >     ASC_SG:   2.237594
[10:32:43] < Debug >     ASC_YK:   1.761268
[10:32:43] < Debug >     B_DIST: -0.09561784
[10:32:43] < General >   Log likelihood (N=268):   -346.404 Gradient norm:      2e+01  
[10:32:43] < Debug >     ASC_MI: -0.1385895
[10:32:43] < Debug >     ASC_OC:  0.3211055
[10:32:43] < Debug >     ASC_RY:    1.29979
[10:32:43] < Debug >     ASC_SC:  0.2310763
[10:32:43] < Debug >     ASC_SG:   2.391478
[10:32:43] < Debug >     ASC_YK:   1.940762
[10:32:43] < Debug >     B_DIST: -0.09964486
[10:32:43] < General >   Log likelihood (N=268):   -345.874 Gradient norm:          9  
[10:32:43] < Debug >     ASC_MI: -0.03119986
[10:32:43] < Debug >     ASC_OC:  0.4806164
[10:32:43] < Debug >     ASC_RY:   1.439033
[10:32:43] < Debug >     ASC_SC:  0.3463005
[10:32:43] < Debug >     ASC_SG:   2.556257
[10:32:43] < Debug >     ASC_

[10:32:43] < Debug >     ASC_MI:  0.9477415
[10:32:43] < Debug >     ASC_OC:  0.9129864
[10:32:43] < Debug >     ASC_RY:    2.13899
[10:32:43] < Debug >     ASC_SC:   1.242035
[10:32:43] < Debug >     ASC_SG:   3.158172
[10:32:43] < Debug >     ASC_YK:   2.627402
[10:32:43] < Debug >     B_DIST: -0.09427838
[10:32:43] < General >   Log likelihood (N=268):  -343.2865 Gradient norm:     0.0009  
[10:32:43] < Debug >     ASC_MI:  0.9477432
[10:32:44] < Debug >     ASC_OC:  0.9129881
[10:32:44] < Debug >     ASC_RY:   2.138991
[10:32:44] < Debug >     ASC_SC:   1.242035
[10:32:44] < Debug >     ASC_SG:   3.158173
[10:32:44] < Debug >     ASC_YK:   2.627404
[10:32:44] < Debug >     B_DIST: -0.0942785
[10:32:44] < General >   Log likelihood (N=268):  -343.2865 Gradient norm:      8e-06  
[10:32:44] < Debug >     ASC_MI:  0.9477432
[10:32:44] < Debug >     ASC_OC:  0.9129881
[10:32:44] < Debug >     ASC_RY:   2.138991
[10:32:44] < Debug >     ASC_SC:   1.242035
[10:32:44] < Debug >     ASC_SG

[10:32:46] < Debug >     ASC_MI:  0.6665609
[10:32:46] < Debug >     ASC_OC: -0.6554466
[10:32:46] < Debug >     ASC_RY:   2.437892
[10:32:46] < Debug >     ASC_SC:  0.6667665
[10:32:46] < Debug >     ASC_SG:   4.151548
[10:32:46] < Debug >     ASC_YK:     2.9459
[10:32:46] < Debug >     B_DIST: -0.05778654
[10:32:46] < General >   Log likelihood (N=851):  -880.8701 Gradient norm:      1e+02  
[10:32:46] < Debug >     ASC_MI:  0.7236648
[10:32:46] < Debug >     ASC_OC: -0.6710958
[10:32:46] < Debug >     ASC_RY:   2.514379
[10:32:46] < Debug >     ASC_SC:  0.7150081
[10:32:46] < Debug >     ASC_SG:   4.215321
[10:32:46] < Debug >     ASC_YK:   2.967004
[10:32:46] < Debug >     B_DIST: -0.05451533
[10:32:46] < General >   Log likelihood (N=851):  -880.5772 Gradient norm:      4e+01  
[10:32:46] < Debug >     ASC_MI:  0.6396728
[10:32:46] < Debug >     ASC_OC: -0.6658462
[10:32:46] < Debug >     ASC_RY:   2.438052
[10:32:46] < Debug >     ASC_SC:  0.6409274
[10:32:46] < Debug >     ASC_S

[10:32:46] < Debug >     ASC_MI:   1.322871
[10:32:46] < Debug >     ASC_OC:    0.98558
[10:32:46] < Debug >     ASC_RY:   3.268097
[10:32:46] < Debug >     ASC_SC:   1.404074
[10:32:46] < Debug >     ASC_SG:   4.965901
[10:32:46] < Debug >     ASC_YK:    3.70838
[10:32:46] < Debug >     B_DIST: -0.05415869
[10:32:46] < General >   Log likelihood (N=851):  -875.7986 Gradient norm:      1e-05 Hessian norm:       2e+04 BHHH norm:       2e+04
[10:32:46] < General >   Results saved in file SMTO_Segmented_Output/SMTO_Campus_Choice_Segment_9.html
[10:32:46] < General >   Results saved in file SMTO_Segmented_Output/SMTO_Campus_Choice_Segment_9.pickle
[10:32:46] < General >   Log file created: SMTO_Segmented_Output/SMTO_Campus_Choice_Segment_9.log
ASC_MI    =	1.32
ASC_OC    =	0.986
ASC_RY    =	3.27
ASC_SC    =	1.4
ASC_SG    =	4.97
ASC_YK    =	3.71
B_DIST    =	-0.0542
           Value   Std err    t-test       p-value  Rob. Std err  Rob. t-test  \
ASC_MI  1.322871  0.569125  2.324393  2.010446e

[10:32:48] < Debug >     ASC_MI: -0.3022543
[10:32:48] < Debug >     ASC_OC:  -1.549207
[10:32:48] < Debug >     ASC_RY: -0.3522701
[10:32:48] < Debug >     ASC_SC: -0.2275434
[10:32:48] < Debug >     ASC_SG:   2.266798
[10:32:48] < Debug >     ASC_YK:   1.748409
[10:32:48] < Debug >     B_DIST: -0.1039546
[10:32:48] < General >   Log likelihood (N=2225):  -2014.687 Gradient norm:      3e+02  
[10:32:48] < Debug >     ASC_MI: -0.2761712
[10:32:48] < Debug >     ASC_OC:  -1.560549
[10:32:48] < Debug >     ASC_RY:  -0.121987
[10:32:48] < Debug >     ASC_SC: -0.2227824
[10:32:48] < Debug >     ASC_SG:   2.357124
[10:32:48] < Debug >     ASC_YK:   1.722207
[10:32:48] < Debug >     B_DIST: -0.09619592
[10:32:48] < General >   Log likelihood (N=2225):  -2006.946 Gradient norm:      5e+02  
[10:32:48] < Debug >     ASC_MI: -0.2553303
[10:32:48] < Debug >     ASC_OC:  -1.610991
[10:32:48] < Debug >     ASC_RY:  0.1322172
[10:32:48] < Debug >     ASC_SC: -0.2242116
[10:32:48] < Debug >     ASC_

[10:32:49] < Debug >     ASC_MI:   1.538727
[10:32:49] < Debug >     ASC_OC:  -0.370636
[10:32:49] < Debug >     ASC_RY:    1.63782
[10:32:49] < Debug >     ASC_SC:   1.862587
[10:32:49] < Debug >     ASC_SG:   3.927186
[10:32:49] < Debug >     ASC_YK:   3.334198
[10:32:49] < Debug >     B_DIST: -0.1027588
[10:32:49] < General >   Log likelihood (N=2225):  -1955.122 Gradient norm:      2e+01  
[10:32:49] < Debug >     ASC_MI:   1.590317
[10:32:49] < Debug >     ASC_OC: -0.2680273
[10:32:49] < Debug >     ASC_RY:   1.687747
[10:32:49] < Debug >     ASC_SC:   1.942966
[10:32:49] < Debug >     ASC_SG:   3.997583
[10:32:49] < Debug >     ASC_YK:   3.405376
[10:32:49] < Debug >     B_DIST: -0.1034028
[10:32:49] < General >   Log likelihood (N=2225):  -1954.863 Gradient norm:      3e+01  
[10:32:49] < Debug >     ASC_MI:   1.670688
[10:32:49] < Debug >     ASC_OC: -0.1571026
[10:32:49] < Debug >     ASC_RY:   1.765045
[10:32:49] < Debug >     ASC_SC:   2.019496
[10:32:49] < Debug >     ASC_S

[10:32:50] < Debug >     ASC_MI:   2.022949
[10:32:50] < Debug >     ASC_OC:  0.3126388
[10:32:50] < Debug >     ASC_RY:   2.058533
[10:32:50] < Debug >     ASC_SC:   2.000081
[10:32:50] < Debug >     ASC_SG:   4.327709
[10:32:50] < Debug >     ASC_YK:   3.742675
[10:32:50] < Debug >     B_DIST: -0.1019938
[10:32:50] < General >   Log likelihood (N=2225):  -1951.656 Gradient norm:      0.001  
[10:32:50] < Debug >     ASC_MI:   2.022948
[10:32:50] < Debug >     ASC_OC:  0.3126372
[10:32:50] < Debug >     ASC_RY:   2.058532
[10:32:50] < Debug >     ASC_SC:   2.000089
[10:32:50] < Debug >     ASC_SG:   4.327709
[10:32:50] < Debug >     ASC_YK:   3.742675
[10:32:50] < Debug >     B_DIST: -0.1019938
[10:32:50] < General >   Log likelihood (N=2225):  -1951.656 Gradient norm:     0.0003  
[10:32:50] < Debug >     ASC_MI:   2.022948
[10:32:50] < Debug >     ASC_OC:  0.3126372
[10:32:50] < Debug >     ASC_RY:   2.058532
[10:32:50] < Debug >     ASC_SC:   2.000089
[10:32:50] < Debug >     ASC_S

[10:32:52] < Debug >     ASC_OC:   1.480632
[10:32:52] < Debug >     ASC_RY:   2.988656
[10:32:52] < Debug >     ASC_SC:  -3.114895
[10:32:52] < Debug >     ASC_SG:   4.590172
[10:32:52] < Debug >     ASC_YK:   4.066935
[10:32:52] < Debug >     B_DIST: -0.09819648
[10:32:52] < General >   Log likelihood (N=106):  -105.3121 Gradient norm:        0.9  
[10:32:52] < Debug >     ASC_MI:  -0.839365
[10:32:52] < Debug >     ASC_OC:   1.522469
[10:32:52] < Debug >     ASC_RY:   3.050514
[10:32:52] < Debug >     ASC_SC:  -3.161589
[10:32:52] < Debug >     ASC_SG:   4.650952
[10:32:52] < Debug >     ASC_YK:   4.113373
[10:32:52] < Debug >     B_DIST: -0.09906689
[10:32:52] < General >   Log likelihood (N=106):  -105.2942 Gradient norm:        0.8  
[10:32:52] < Debug >     ASC_MI: -0.7491938
[10:32:52] < Debug >     ASC_OC:   1.648159
[10:32:52] < Debug >     ASC_RY:    3.21833
[10:32:52] < Debug >     ASC_SC:  -3.294525
[10:32:52] < Debug >     ASC_SG:    4.81747
[10:32:52] < Debug >     ASC_Y

[10:32:52] < Debug >     ASC_MI:   13.88595
[10:32:52] < Debug >     ASC_OC:   14.81358
[10:32:52] < Debug >     ASC_RY:   16.49561
[10:32:52] < Debug >     ASC_SC:  -14.78865
[10:32:52] < Debug >     ASC_SG:   18.09201
[10:32:52] < Debug >     ASC_YK:   17.56956
[10:32:52] < Debug >     B_DIST: -0.09494674
[10:32:52] < General >   Log likelihood (N=106):  -104.0578 Gradient norm:      0.007  
[10:32:52] < Debug >     ASC_MI:   14.18897
[10:32:52] < Debug >     ASC_OC:   15.11685
[10:32:52] < Debug >     ASC_RY:   16.79873
[10:32:52] < Debug >     ASC_SC:  -15.05078
[10:32:52] < Debug >     ASC_SG:   18.39516
[10:32:52] < Debug >     ASC_YK:   17.87252
[10:32:52] < Debug >     B_DIST: -0.09495375
[10:32:52] < General >   Log likelihood (N=106):  -104.0577 Gradient norm:      0.003  
[10:32:52] < Debug >     ASC_MI:   14.63442
[10:32:52] < Debug >     ASC_OC:   15.56233
[10:32:52] < Debug >     ASC_RY:    17.2443
[10:32:52] < Debug >     ASC_SC:  -15.43605
[10:32:52] < Debug >     ASC_S

[10:32:52] < General >   Results saved in file SMTO_Segmented_Output/SMTO_Campus_Choice_Segment_11.html
[10:32:52] < General >   Results saved in file SMTO_Segmented_Output/SMTO_Campus_Choice_Segment_11.pickle
[10:32:52] < General >   Log file created: SMTO_Segmented_Output/SMTO_Campus_Choice_Segment_11.log
ASC_MI    =	23.3
ASC_OC    =	24.2
ASC_RY    =	25.9
ASC_SC    =	-22.9
ASC_SG    =	27.5
ASC_YK    =	27
B_DIST    =	-0.095
            Value        Std err         t-test   p-value   Rob. Std err  \
ASC_MI  23.289563   1.141991e+05   2.039382e-04  0.999837   1.604277e+03   
ASC_OC  24.216601   1.141991e+05   2.120559e-04  0.999831   1.661404e+03   
ASC_RY  25.898560   1.141991e+05   2.267842e-04  0.999819   1.553068e+03   
ASC_SC -22.920661  1.797693e+308 -1.275004e-307  1.000000  1.797693e+308   
ASC_SG  27.495011   1.141991e+05   2.407637e-04  0.999808   1.663274e+03   
ASC_YK  26.972314   1.141991e+05   2.361867e-04  0.999812   1.667009e+03   
B_DIST  -0.094952   2.550929e-02  -3.72

[10:32:54] < Debug >     ASC_RY:  0.3761717
[10:32:54] < Debug >     ASC_SC:  -1.110123
[10:32:54] < Debug >     ASC_SG:   2.708955
[10:32:54] < Debug >     ASC_YK:   2.036386
[10:32:54] < Debug >     B_DIST: -0.07135081
[10:32:54] < General >   Log likelihood (N=233):  -246.0783 Gradient norm:      2e+01  
[10:32:54] < Debug >     ASC_MI:  -1.591431
[10:32:54] < Debug >     ASC_OC: -0.4640098
[10:32:54] < Debug >     ASC_RY:   0.647685
[10:32:54] < Debug >     ASC_SC:  -1.147971
[10:32:54] < Debug >     ASC_SG:   2.632859
[10:32:54] < Debug >     ASC_YK:   2.015737
[10:32:54] < Debug >     B_DIST: -0.06758366
[10:32:54] < General >   Log likelihood (N=233):  -243.7354 Gradient norm:      2e+01  
[10:32:54] < Debug >     ASC_MI:  -1.710806
[10:32:54] < Debug >     ASC_OC: -0.3115394
[10:32:54] < Debug >     ASC_RY:  0.9198199
[10:32:54] < Debug >     ASC_SC:  -1.211345
[10:32:54] < Debug >     ASC_SG:   2.678986
[10:32:54] < Debug >     ASC_YK:   2.015162
[10:32:54] < Debug >     B_DIS

[10:32:54] < Debug >     ASC_MI:  -5.419198
[10:32:54] < Debug >     ASC_OC:   1.089055
[10:32:54] < Debug >     ASC_RY:   1.789629
[10:32:54] < Debug >     ASC_SC: -0.5576171
[10:32:54] < Debug >     ASC_SG:   3.691025
[10:32:54] < Debug >     ASC_YK:   2.911087
[10:32:54] < Debug >     B_DIST: -0.06101184
[10:32:54] < General >   Log likelihood (N=233):  -238.3169 Gradient norm:        0.4  
[10:32:54] < Debug >     ASC_MI:  -5.755993
[10:32:54] < Debug >     ASC_OC:   1.075501
[10:32:54] < Debug >     ASC_RY:   1.772337
[10:32:54] < Debug >     ASC_SC: -0.6106892
[10:32:54] < Debug >     ASC_SG:   3.672571
[10:32:54] < Debug >     ASC_YK:   2.885797
[10:32:54] < Debug >     B_DIST: -0.06072932
[10:32:54] < General >   Log likelihood (N=233):  -238.3091 Gradient norm:        0.4  
[10:32:54] < Debug >     ASC_MI:  -6.590795
[10:32:54] < Debug >     ASC_OC:   1.085789
[10:32:54] < Debug >     ASC_RY:   1.658135
[10:32:54] < Debug >     ASC_SC: -0.8289563
[10:32:54] < Debug >     ASC_S

[10:32:54] < Debug >     ASC_RY:   1.762592
[10:32:54] < Debug >     ASC_SC: -0.6843811
[10:32:54] < Debug >     ASC_SG:   3.668023
[10:32:54] < Debug >     ASC_YK:   2.880114
[10:32:54] < Debug >     B_DIST: -0.06060838
[10:32:54] < General >   Log likelihood (N=233):  -238.2983 Gradient norm:        0.2  
[10:32:54] < Debug >     ASC_MI:   -15.0543
[10:32:54] < Debug >     ASC_OC:   1.070277
[10:32:54] < Debug >     ASC_RY:   1.762187
[10:32:54] < Debug >     ASC_SC: -0.6852318
[10:32:54] < Debug >     ASC_SG:   3.667527
[10:32:54] < Debug >     ASC_YK:   2.879501
[10:32:54] < Debug >     B_DIST: -0.06057436
[10:32:54] < General >   Log likelihood (N=233):  -238.2983 Gradient norm:       0.02  
[10:32:54] < Debug >     ASC_MI:  -15.59709
[10:32:54] < Debug >     ASC_OC:   1.070352
[10:32:54] < Debug >     ASC_RY:   1.762273
[10:32:54] < Debug >     ASC_SC: -0.6852449
[10:32:54] < Debug >     ASC_SG:    3.66761
[10:32:54] < Debug >     ASC_YK:    2.87955
[10:32:54] < Debug >     B_DIS

[10:32:54] < General >   Results saved in file SMTO_Segmented_Output/SMTO_Campus_Choice_Segment_12.html
[10:32:54] < General >   Results saved in file SMTO_Segmented_Output/SMTO_Campus_Choice_Segment_12.pickle
[10:32:54] < General >   Log file created: SMTO_Segmented_Output/SMTO_Campus_Choice_Segment_12.log
ASC_MI    =	-23.6
ASC_OC    =	1.07
ASC_RY    =	1.76
ASC_SC    =	-0.685
ASC_SG    =	3.67
ASC_YK    =	2.88
B_DIST    =	-0.0606
            Value       Std err    t-test       p-value  Rob. Std err  \
ASC_MI -23.648202  84963.240995 -0.000278  9.997779e-01      0.579407   
ASC_OC   1.070242      0.653991  1.636478  1.017396e-01      0.650781   
ASC_RY   1.762173      0.618126  2.850832  4.360498e-03      0.617966   
ASC_SC  -0.685325      1.156813 -0.592425  5.535660e-01      1.156292   
ASC_SG   3.667503      0.586227  6.256116  3.946821e-10      0.585351   
ASC_YK   2.879451      0.597897  4.815965  1.464900e-06      0.597874   
B_DIST  -0.060570      0.016723 -3.621862  2.924898e-04

In [9]:
pandasResults

Value       Std err    t-test       p-value  Rob. Std err  \
ASC_MI -23.648202  84963.240995 -0.000278  9.997779e-01      0.579407   
ASC_OC   1.070242      0.653991  1.636478  1.017396e-01      0.650781   
ASC_RY   1.762173      0.618126  2.850832  4.360498e-03      0.617966   
ASC_SC  -0.685325      1.156813 -0.592425  5.535660e-01      1.156292   
ASC_SG   3.667503      0.586227  6.256116  3.946821e-10      0.585351   
ASC_YK   2.879451      0.597897  4.815965  1.464900e-06      0.597874   
B_DIST  -0.060570      0.016723 -3.621862  2.924898e-04      0.017145   

        Rob. t-test  Rob. p-value  
ASC_MI   -40.814486  0.000000e+00  
ASC_OC     1.644549  1.000629e-01  
ASC_RY     2.851570  4.350388e-03  
ASC_SC    -0.592692  5.533872e-01  
ASC_SG     6.265474  3.716929e-10  
ASC_YK     4.816153  1.463526e-06  
B_DIST    -3.532831  4.111349e-04